pkg/drivers/commonconfig_test.go

```diff
@@ -129,9 +129,15 @@ func TestGetNode(t *testing.T) {
 	ctx := context.Background()
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			_, err := GetNode(ctx, tt.csm, config, tt.driverName, tt.filename)
+			node, err := GetNode(ctx, tt.csm, config, tt.driverName, tt.filename)
 			if tt.expectedErr == "" {
 				assert.Nil(t, err)
+				initcontainers := node.DaemonSetApplyConfig.Spec.Template.Spec.InitContainers
+				for i := range initcontainers {
+					if *initcontainers[i].Name == "mdm-container" {
```

Comment:
It would be better to require that the mdm-container is present among the init containers of the pflex driver.

Reply:
added a check for this